Configuration/EventContent/python/EventContent_cff.py

```diff
@@ -885,39 +835,31 @@ def SwapKeepAndDrop(l):
 for _entry in [MINIAODEventContent, MINIAODSIMEventContent]:
     fastSim.toModify(_entry, outputCommands = _entry.outputCommands + fastSimEC.dropPatTrigger)
 
+from Configuration.Eras.Modifier_run2_GEM_2017_cff import run2_GEM_2017
```

Comment:
All these phase2, gem, HI modifications should be better moved right below the corresponding definitions of the event content, as it is done already for hgcal and pps (for example)